ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentActionMenu/DocumentActionMenu.js

```diff
@@ -93,59 +90,91 @@ export default class DocumentActionMenu extends Component {
 
         <Menu pointing secondary vertical fluid className="left">
           <Menu.Item
-            name="header"
-            active={activeItem === 'header'}
-            onClick={(e, { name }) => this.scrollTo(anchors.summaryRef, name)}
+            name="metadata"
+            active={activeItem === 'metadata'}
+            activeClass="active"
+            as={ScrollLink}
+            to={'metadata'}
+            spy={true}
+            onSetActive={() => this.setState({ activeItem: 'metadata' })}
+            offset={offset}
```

Comment:
These menu items are super similar. They can be created programmatically somehow like this
```js
[
  {name: "metadata", label: "Metadata"},
  {name: "loan-requests", label: "loan-requests"},
].map(item => <Menu.Item name={item.name} spy={true} ... >{item.name}</Menu.Item>)

```

Reply:
I can make them as a reusable component, I think it is more "react way", I will do it in my next PR.